crunchy-postgres/charts/crunchy-postgres/templates/PostgresCluster.yaml

```diff
@@ -87,16 +87,41 @@ spec:
       {{ if .Values.pgBackRest.image }}
       image: {{ .Values.pgBackRest.image }}
       {{ end }}
+      configuration:
+        - secret:
+            name: {{ .Values.pgBackRest.repos.configuration.secretName }}
       global:
         # hardcoding repo1 until we solution allowing multiple repos
         repo1-retention-full: {{ .Values.pgBackRest.retention | quote }}
         repo1-retention-full-type: {{ .Values.pgBackRest.retentionFullType }}
 
+        # how many differential backups to keep
+        repo1-retention-diff: {{ .Values.pgBackRest.retentionDifferential | quote}}
+
+        #Whether to expire .wal files on any backup or full backup only
+        repo1-retention-archive-type: {{ .Values.pgBackRest.retentionArchiveType }}
```

Comment:
What does this do? Docs recommend that this would usually be left at the default, which is full.